kalite/distributed/static/js/distributed/content/views.js

```diff
@@ -136,6 +136,10 @@ window.ContentBaseView = BaseView.extend({
         this.log_model.set({
             points: Math.min(this.possible_points, Math.floor(this.possible_points * progress))
         });
+
```

Comment:
No need to trigger two change events here, can pass the progress: progress key: value pair in the previous .set method call.